config/database.php

```diff
@@ -55,7 +55,7 @@
             'password' => env('DB_PASSWORD', ''),
             'unix_socket' => env('DB_SOCKET', ''),
             'charset' => env('DB_CHARSET', 'utf8mb4'),
-            'collation' => env('DB_COLLATION', 'utf8mb4_unicode_ci'),
+            'collation' => env('DB_COLLATION', 'uca1400_ai_ci'),
```

Comment:
I tried recreating the test db, and got this error:
```
SQLSTATE[HY000]: General error: 1273 Unknown collation: 'uca1400_ai_ci' (SQL: SHOW FULL TABLES WHERE table_type = 'BASE TABLE')
```

fwiw, I'm on MariaDB 10.4:
```
Server version: 10.4.28-MariaDB Source distribution
```

Reply:
oh, that'd be a problem, yeah. docker container and server are using 10.11.
you can set the collation to unicode in .env in the meantime.